VideoEncoding/Encoding_OverlayImage/Program.cs

```diff
@@ -1,643 +1,524 @@
+    // In production code, you may want to be cautious about that. It really depends on your scenario.
+    var transform = await mediaServicesAccount.GetMediaTransforms().CreateOrUpdateAsync(
+        WaitUntil.Completed,
+        transformName,
+        new MediaTransformData
         {
-            // This code does not check if Transform already exists, so it will overwrite existing one. 
-            // That is good for when you are testing things out and getting the layout of the overlay into the right position.
-            // Transform.CreateOrUpdate() will overwrite any existing Transform with the same name.
-
-            Console.WriteLine("Creating a custom transform...");
-
-            // Create a new Transform Outputs array - this defines the set of outputs for the Transform
-            var outputs = new TransformOutput[]
+            Outputs =
             {
-                // Create a new TransformOutput with a custom Standard Encoder Preset and overlay
-                new TransformOutput
-                {
-                    Preset = new StandardEncoderPreset
-                    {
-                        Filters = new Filters
+                // Create a new TransformOutput with a custom Standard Encoder Preset using the HEVC (H265Layer) codec
+                // This demonstrates how to create custom codec and layer output settings
+                new MediaTransformOutput(
+                    preset: new StandardEncoderPreset(
+                        codecs: new MediaCodecBase[]
                         {
-                            Overlays = new List<Overlay>
-                            {
-                                new VideoOverlay
-                                {
-                                    InputLabel = OverlayLabel,   // same as the one used in the JobInput to identify which asset is the overlay image
-                                    Position = new Rectangle( "1200","670"), // left, top position of the overlay in absolute pixel position relative to the source videos resolution. 
-                                    // Percentage based settings are coming soon, but not yet supported. In the future you can set this to "90%","90%" for example to be resolution independent on the source video positioning.
-                                    // Opacity = 0.25  // opacity can be adjusted on the overlay.
-                                }
-                            }
-                        },
-                        Codecs = new List<Codec>
-                        {
-                            new AacAudio
-                            {
-                            },
+                            // Add an AAC Audio layer for the audio encoding
+                            new AacAudio(),
+                            // Next, add a HEVC (H.265) for the video encoding
```

Comment:
H.264 instead

Reply:
I've updated the code to resolve this issue.